Assets/Scripts/Player/PlayerController.cs

```diff
@@ -114,26 +132,101 @@ private void SetUpControls()
         private void TogglePause(InputAction.CallbackContext obj) => GameManager.Instance.TogglePause();
 
         /// <summary>
-        /// When the right mouse button is pressed, the player moves to the pressed location using a raycast and the NavMeshAgent.
+        /// Is called on mouse click.
+        /// If right clicked, checks whether clicked on a damageable object or the ground.
+        /// If clicked on a damageable object, attack it. If clicked on the ground, cancel attack
+        /// and move to the clicked point.
         /// </summary>
-        private void Move(InputAction.CallbackContext obj)
+        private void OnRightClick(InputAction.CallbackContext obj)
         {
             Ray clickRay = _camera.ScreenPointToRay(Mouse.current.position.ReadValue());
 
-            // Get ground position from mouse click
-            if (Physics.Raycast(clickRay, out RaycastHit hit, 10000, moveClickLayers))
+            // only change target / move, if not performing a hit
+            if (Physics.Raycast(clickRay, out RaycastHit hit, 10000) && _attackLogic.Status == AttackLogic.AttackStatus.None)
```

Comment:
Would maybe change this in the future to make it possible to cancel attacks by moving away from the enemy. Doesn't really feel like the player is in control like this 😬

Reply:
We can discuss about that. It might be useful, idk. It's a tiny change howsoever.